hypothesis_protobuf/module_conversion.py

```diff
@@ -9,7 +9,7 @@
 from google.protobuf.internal.well_known_types import FieldDescriptor
 
 
-SINGLEPRECISION = dict(max_value=(2 - 2 ** -23) * 2 ** 127, min_value=-(2 - 2 ** -23) * 2 ** 127)
+SINGLEPRECISION = dict(max_value=(2 - 2 ** -23) * 2 ** 127, min_value=-(2 - 2 ** -23) * 2 ** 127, width=32)
```

Comment:
Do we need to keep all the other parameters. If width=32 generates a single-precision floating point number, it should be within range.